pkg/gateway/lb/handler/grpc/handler.go

```diff
@@ -1561,7 +1577,11 @@ func (s *server) Remove(ctx context.Context, req *payload.Remove_Request) (locs
 			}
 			return nil, err
 		}
-		req.Config.SkipStrictExistCheck = true
+		if req.GetConfig() != nil {
+			req.Config.SkipStrictExistCheck = true
+		} else {
+			req.Config = &payload.Remove_Config{SkipStrictExistCheck: true}
+		}
 	}
 	var mu sync.Mutex
 	locs = &payload.Object_Location{
```

Comment:
same, is there no need to modify MultiRemove?

Reply:
thank you for review, I fixed it.